src/Blocks/components/input/components/input-editor.js

```diff
@@ -23,29 +23,39 @@ export const InputEditor = (attributes) => {
 	const inputName = checkAttr('inputName', attributes, manifest);
 	const inputValue = checkAttr('inputValue', attributes, manifest);
 	const inputPlaceholder = checkAttr('inputPlaceholder', attributes, manifest);
-	let inputType = checkAttr('inputType', attributes, manifest);
+	const inputType = checkAttr('inputType', attributes, manifest);
+	const inputMin = checkAttr('inputMin', attributes, manifest);
+	const inputMax = checkAttr('inputMax', attributes, manifest);
+	const inputStep = checkAttr('inputStep', attributes, manifest);
 
 	preventSaveOnMissingProps(blockClientId, getAttrKey('inputName', attributes, manifest), inputName);
 
-	// For some reason React won't allow input type email.
-	if (inputType === 'email' || inputType === 'url') {
-		inputType = 'text';
-	}
-
 	const inputClass = classnames([
 		selector(componentClass, componentClass),
 		selector(additionalClass, additionalClass),
 	]);
```

Comment:
```suggestion
	const inputClass = classnames(componentClass, additionalClass);
```
`classnames` from the `classnames` package is not tree-shakeable, that's why the FE libs version is preferred.